### DISCUSSION TITLE
Menu buttons registering clicks when hovering over osu! cookie and right-clicking

### DISCUSSION THREAD

**Discussion by eggooooo** [ppy/osu#18036] [Q&A]
When hovering over the osu! cookie and right-clicking, the mouse down (e.g brightness increase) animation gets triggered.

This isn't specifically happening only when right-clicking, but rather any mouse button that isnt left-click. Just used for reference

https://user-images.githubusercontent.com/64386789/166136800-e8087ebc-f61e-4519-b3de-8dd255d09863.mp4

**Comment by smoogipoo** [comment]
Is this a regression from the `DragBlocksClick` override? Would be unfortunate if so.

**Reply by peppy** [reply]
It's not a regression. This has always been how it worked. I'm not even sure this is an issue.
